packages/pdf-generator/package.json

```diff
@@ -34,8 +34,8 @@
         "@patternfly/react-styles": "^3.6.26",
         "@patternfly/react-tokens": "^2.7.25",
         "@react-pdf/renderer": "^1.6.8",
-        "react": "16.12.0",
-        "react-dom": "16.12.0",
+        "react": "^16.12.0",
```

Comment:
Wasn't there some issue with a higher version if react/react-dom that made you lock them? I vaguely remember that there was some problem with async rendering.

Reply:
Oh, yeah. Mhh gotta use the locked version... It'll bload the bundle by 900 KB unfortunatelly.

Reply:
Yeah :cry: sorry about reminding you.